azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTestBase.java

```diff
@@ -170,4 +171,8 @@ protected void succeedJobs(final String... jobs) {
     }
   }
 
+  protected void assertEvents(final Type... types) {
```

Comment:
I think it makes the test cases easier to read. Same pattern is already used for assertStatus. I don't see it as a problem that you need to navigate to the superclass if you want to see how it's implemented. That isn't much different than navigating to the method if it was at the bottom of the concrete test class - at least if using an IDE, like one usually does.